templates/base.html

```diff
@@ -11,12 +11,26 @@
 
 <body>
   <div class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-white border-bottom shadow-sm">
-    <h5 class="my-0 mr-md-auto font-weight-normal">Student Tracker</h5>
+    <h5 class="my-0 mr-md-auto font-weight-normal"><a href="{% url "home" %}"> Student Tracker</a></h5>
+   {% block nav_links%}
     <nav class="my-2 my-md-0 mr-md-3">
       <a class="p-2 text-dark" href="{% url "student-list" %}">Students</a>
       <a class="p-2 text-dark" href="{% url "standard-list" %}">Standards</a>
       <a class="p-2 text-dark" href="{% url "assessment-list" %}">Assessments</a>
     </nav>
+    <div>
+      {% if user.is_authenticated %}
+        <a href="{% url "signout" %}" class="btn btn-primary btn-sm ml-2">
+          Sign Out
+          </a>
+      {% else %}
+        <a href="{% url "login" %}" class="btn btn-primary btn-sm ml-2">
+        Login
+        </a>
+      {% endif %}
```

Comment:
Nice! As a design choice, you could also hide the links above if you're not signed in, but that's definitely not necessary.